rust/lancedb/src/database/listing.rs

```diff
@@ -678,6 +679,65 @@ impl Database for ListingDatabase {
         }
     }
 
+    async fn clone_table(&self, request: CloneTableRequest) -> Result<Arc<dyn BaseTable>> {
+        if !request.target_namespace.is_empty() {
+            return Err(Error::NotSupported {
+                message: "Namespace parameter is not supported for listing database. Only root namespace is supported.".into(),
```

Comment:
This error message seems to be wrong. What's the relation of this API with `listing database`?